plugins/chat/assets/javascripts/discourse/components/chat-thread-pane.js

```diff
@@ -0,0 +1,20 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+
+export default class ChatThreadPane extends Component {
+  @service siteSettings;
+  @service currentUser;
+  @service chat;
+  @service router;
+
+  channel = null;
+  thread = null;
+
+  @action
+  closeThread() {
+    return this.router.transitionTo("chat.channel", {
```

Comment:
Not sure if this is correct, would be nice if this did not reload the whole channel.